config/env_sample.hjson

```diff
@@ -94,7 +94,7 @@
     "GOOGLE_APPLICATION_CREDENTIALS": "/secrets/bq_cred.json",
     # LTI Configuration
     # LTI is disabled by default
```

Comment:
Could change this comment to "off by default" or "deactivated by default"

Reply:
Nooo! It still snuck in there. Or can change it to "not enabled" to be consistent with the name.

Reply:
So it looks like there's two other properties that are "DISABLED": `LTI_CONFIG_DISABLE_DEPLOY_ID_VALIDATION `and `VIEWS_DISABLED`, along with other comments.

I think this would have to be a bigger refactor if we wanted to change that.